content/en/community.md

```diff
@@ -19,6 +19,12 @@ The following are ways to engage directly with the SciPy project and community.
 _Please note that we encourage users and community members to support each
 other for usage questions - see [Get Help](/gethelp)._
 
+### [SciPy community meetings](https://scientific-python.org/calendars/)
+
+The Scientific Python community is organizing various meetings. SciPy
```

Comment:
This first sentence is confusing, I suggest to just delete it. The SciPy community meeting is organized by the SciPy project, not The Scientific Python community

Reply:
Ok sure.